cluster-api/providers/gcp/go.mod

```diff
@@ -1,33 +1,34 @@
 module openshift/installer/cluster-api/providers/gcp
 
-go 1.22.0
+go 1.23.7
 
-toolchain go1.23.4
+toolchain go1.24.2
 
 // CAPG replace block: these are taken directly from
 // https://github.com/kubernetes-sigs/cluster-api-provider-gcp/blob/main/go.mod
 // & should stay in sync with upstream.
 replace (
 	// cel-go should match the version imported by CAPI.
-	github.com/google/cel-go => github.com/google/cel-go v0.20.1
+	github.com/google/cel-go => github.com/google/cel-go v0.22.0
 	// kube-openapi should match the version imported by CAPI.
 	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20240228011516-70dd3763d340
-	sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.9.4
+	sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.10.3
 )
 
-require sigs.k8s.io/cluster-api-provider-gcp v1.8.1-0.20250225090028-d80bfabadd3f
+require sigs.k8s.io/cluster-api-provider-gcp v1.10.1-0.20251016125449-c2c9fb21d6e5
```

Comment:
Are we OK to bump this version one to match the commit version pinned on `main` for the top-level `go.mod`?

https://github.com/openshift/installer/blob/4180662515e75d11d0a7984c66d574f3c64a765a/go.mod#L142